OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -2377,6 +2377,17 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 						node.Key = "ParticleSize";
 				}
 
+				// Set ChargeDelay to 300 by default
+				if (engineVersion < 20151111)
+				{
+					if (depth == 1 && node.Key == "PortableChrono")
+					{
+						var delay = node.Value.Nodes.FirstOrDefault(n => n.Key == "ChargeDelay");
+						if (delay == null)
+							node.Value.Nodes.Add(new MiniYamlNode("ChargeDelay", "300"));
```

Comment:
But now there is a difference between the engine fallback and the value you're inserting into yaml.